src/assets/js/pages/EventPage/tabs/ActionsTab.js

```diff
@@ -28,6 +28,21 @@ export default class ActionsTab extends React.Component {
       });
   }
 
+  generateTeams = () => {
+    let eventAlias = this.props.event.alias;
+    exportUsers(eventAlias)
```

Comment:
Great start - doesn't look like you're actually calling the API `generateTeams` though?

Reply:
Oops haha